substrate/frame/executive/src/tests.rs

```diff
@@ -355,19 +394,46 @@ impl frame_support::traits::Get<sp_version::RuntimeVersion> for RuntimeVersion {
 	}
 }
 
+#[derive(Clone, Debug, Encode, codec::Decode, PartialEq, Eq, scale_info::TypeInfo)]
+pub struct AccountU64(u64);
+impl sp_runtime::traits::IdentifyAccount for AccountU64 {
+	type AccountId = u64;
+	fn into_account(self) -> u64 {
+		self.0
+	}
+}
+
+impl sp_runtime::traits::Verify for AccountU64 {
+	type Signer = AccountU64;
+	fn verify<L: sp_runtime::traits::Lazy<[u8]>>(
+		&self,
+		_msg: L,
+		_signer: &<Self::Signer as sp_runtime::traits::IdentifyAccount>::AccountId,
+	) -> bool {
+		true
+	}
+}
+
+impl From<u64> for AccountU64 {
+	fn from(value: u64) -> Self {
+		Self(value)
+	}
+}
+
 parameter_types! {
 	pub static RuntimeVersionTestValues: sp_version::RuntimeVersion =
 		Default::default();
 }
 
-type SignedExtra = (
+type TxExtension = (
```

Comment:
Okay, this is one of the things that I saw a lot of students struggle with in PBA as well, and I think it is a fair ask: We have no content on how to test a signed extension. No one knew hot to start, or where to look Probably the only example to learn it is to look here, or in one pallet that has tx-extensions.

A good action plan IMO is: 

1. remove this `WatchDummy` from `examples/basics` and move it to a new `examples/tx-extension`, with more examples. 
2. Provide and explain a testing setup there. This probably means something like `AccountU64` should move to `sp_runtime::testing` as it is useful in multiple pallets. 
3. link to this `examples/tx-extenion` from `reference_docs::tx_extension`.

If you agree, please make an issue for this as a follow-up.

Reply:
I think the objectives of this comment were met with the introduction of `substrate/frame/examples/authorization-tx-extension`. Let me know if it's good enough.